client/components/domains/domain-upsell-calout/index.jsx

```diff
@@ -0,0 +1,80 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { isFreePlanProduct } from '@automattic/calypso-products';
+import { Gridicon } from '@automattic/components';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import page from 'page';
+import { useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { isCurrentUserEmailVerified } from 'calypso/state/current-user/selectors';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference, hasReceivedRemotePreferences } from 'calypso/state/preferences/selectors';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+import './style.scss';
+
+const DomainUpsellCalout = ( { trackEvent } ) => {
+	const { __ } = useI18n();
+	const dispatch = useDispatch();
+	const site = useSelector( ( state ) => getSelectedSite( state ) );
+	const trackEventView = `calypso_${ trackEvent }_view`;
+	const trackEventClick = `calypso_${ trackEvent }_click`;
+	const trackEventDismiss = `calypso_${ trackEvent }_dismiss`;
+	const dismissPreference = `${ trackEvent }-${ site.ID }`;
+	const isEmailVerified = useSelector( ( state ) => isCurrentUserEmailVerified( state ) );
+	const siteDomains = useSelector( ( state ) => getDomainsBySiteId( state, site.ID ) );
+	const hasPreferences = useSelector( hasReceivedRemotePreferences );
+	const isDismissed = useSelector( ( state ) => getPreference( state, dismissPreference ) );
+
+	const getCtaClickHandler = useCallback( () => {
+		recordTracksEvent( trackEventClick );
+		page( sprintf( '/domains/add/%s?domainAndPlanPackage=true', site.domain ) );
```

Comment:
`sprintf`s are mainly used for translations that have placeholders in them to help the translators. This isn't our use case here. I think something like this would work (not tested):
```suggestion
		page( `/domains/add/${site.domain}?domainAndPlanPackage=true`);
```